service/src/main/java/com/zebrunner/reporting/service/integration/tool/adapter/notificationservice/SlackIntegrationAdapter.java

```diff
@@ -0,0 +1,139 @@
+package com.zebrunner.reporting.service.integration.tool.adapter.notificationservice;
+
+import com.zebrunner.reporting.domain.db.TestRun;
+import com.zebrunner.reporting.domain.entity.integration.Integration;
+import com.zebrunner.reporting.service.integration.tool.adapter.AbstractIntegrationAdapter;
+import com.zebrunner.reporting.service.integration.tool.adapter.AdapterParam;
+import com.zebrunner.reporting.service.integration.tool.impl.AutomationServerService;
+import in.ashwanthkumar.slack.webhook.SlackAttachment;
```

Comment:
Side note: i think we should consider switch to direct HTTP communication with Slack instead of this lib

Reply:
I'll create the ticket for it